act-rules-format.bs

```diff
@@ -64,7 +64,7 @@ An ACT Rule MUST have a description that is in plain language and provides a bri
 Accessibility Requirements {#structure-accessibility-requirements}
 -------------------------------------------------------------------
 
-An ACT Rule MUST identify the accessibility requirements to which the rule maps. For example, WCAG 2.0 Success Criterion 1.1.1. An ACT Rule is a complete or partial test for one or more accessibility requirements.
+An ACT Rule MUST identify the accessibility requirements to which the rule maps. For example, [WCAG 2.0 Success Criterion 1.1.1](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#text-equiv-all) [[WCAG20]]. An ACT Rule is a complete or partial test for one or more accessibility requirements.
```

Comment:
Not certain about this, but would it be better to link to the "How to Meet WCAG 2.0" quick reference guide instead of the static TR doc? https://www.w3.org/WAI/WCAG20/quickref/#text-equiv-all

Reply:
Let's raise this for discussion on Thursday's call. Thanks, Moe.